Makefile

```diff
@@ -263,6 +265,10 @@ cluster-templates-v1beta1: $(KUSTOMIZE) ## Generate cluster templates for v1beta
 	$(KUSTOMIZE) build $(NUTANIX_E2E_TEMPLATES)/v1beta1/cluster-template-no-nmt --load-restrictor LoadRestrictionsNone > $(NUTANIX_E2E_TEMPLATES)/v1beta1/cluster-template-no-nmt.yaml
 	$(KUSTOMIZE) build $(NUTANIX_E2E_TEMPLATES)/v1beta1/cluster-template-project --load-restrictor LoadRestrictionsNone > $(NUTANIX_E2E_TEMPLATES)/v1beta1/cluster-template-project.yaml
 
+cluster-flavors: $(KUSTOMIZE) ## Generate cluster flavors for all versions
```

Comment:
we should add dependency of this on cluster-templates-v1beta1 as well or merge them

Reply:
as eventually we will need them there in e2e for testing purposes as well

Reply:
ok let's make a dependency because e2e flavor and release flavors are not the same